src/sentry/models/integration.py

```diff
@@ -74,7 +74,16 @@ class ExternalProviders(Enum):
 }
 
 
-class ExternalTeam(DefaultFieldsModel):
+class ExternalProviderMixin:
+    def get_provider_string(provider_int):
+        return EXTERNAL_PROVIDERS.get(ExternalProviders(provider_int), "unknown")
```

Comment:
@NisanthanNanthakumar Hmmm if we get an unknown provider, shouldn't we have some sort of exception?

Reply:
@scefali so my thought process was to return this value bc this will only be hit during the getList. But when they try to update the record, the serializer validation will fail on `provider: "unknown"`. So they'll update with the correct provider. There is a test for this https://github.com/getsentry/sentry/blob/master/tests/sentry/api/endpoints/test_external_team_details.py#L33

Reply:
@NisanthanNanthakumar cool, as long as we check this at the API level we are good